config/images/images.yaml

```diff
@@ -216,6 +216,16 @@ images:
   - 0.127.0
   - 0.129.1
   - 0.139.0
+- source: ghcr.io/open-telemetry/opentelemetry-operator/target-allocator
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/opentelemetry-operator/target-allocator
+  tags:
+  - v0.140.0
+  - v0.141.0
+- source: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/opentelemetry-operator/opentelemetry-collector
+  tags:
+  - 0.140.0
+  - 0.141.0
```

Comment:
Do we need this or can we just fall back to always using the `contrib` images (which also seems to have support for extensions that we might need (bearer token auth))?